ledger/sandbox/src/main/scala/com/digitalasset/platform/apiserver/services/ApiSubmissionService.scala

```diff
@@ -55,6 +57,9 @@ object ApiSubmissionService {
 
   type RecordUpdate = Either[LfError, (Transaction, BlindingInfo)]
 
+  val MetricPrefix: MetricName =
```

Comment:
Why not keeping it in the more restricted scope?

Reply:
Gonna move it back. I wanted to use it for execution stuff but I changed my mind.